### DISCUSSION TITLE
-1 dollar fare difference

### DISCUSSION THREAD

**Discussion by jwin3** [jdholtz/auto-southwest-check-in#102] [General]
I know this isn't an issue with the script and more of a southwest issue, but sometimes I get a notification that a fare is 1 dollar lower than my current fare. So I go to the southwest website and the website will say there's a fare difference of -1 dollar, I go through to change it, but the fare is the same exact thing with no credit. IF this is a false positive that other people are getting as well, can we ignore 1 dollar fare diffrences so we don't get a notification each time?

**Comment by jdholtz** [comment]
Yes, I can make sure the fare difference is greater than one dollar before sending the notification

**Reply by jdholtz** [reply]
This is added in the latest commit of the `develop` branch (0b43a44).
